reproducibility_FAQs.md

```diff
@@ -0,0 +1,253 @@
+﻿## Frequently Asked Questions: Reproducibility Packages
+
+Before diving into the FAQs, we recommend reviewing the [Guidance Note](https://github.com/worldbank/wb-reproducible-research-repository/blob/main/guidance_note_wb.md) for a comprehensive overview of reproducibility standards and submission expectations.
+
+**Note for World Bank Staff:** For internal FAQs tailored to World Bank researchers—please visit the [internal reproducibility resources page](https://worldbankgroup.sharepoint.com/sites/WBPRWP/SitePages/PublishingPages/FAQs-10262018-142102.aspx).
+
+## Table of Contents
+
+- [General FAQs](#general-faqs)
+  - [How do I submit a package?](#how-do-i-submit-a-package)
+  - [How long does the reproducibility verification process take?](#how-long-does-the-reproducibility-verification-process-take)
+  - [How should I organize my package?](#how-should-i-organize-my-package)
+  - [What does the reproducibility report contain?](#what-does-the-reproducibility-report-contain)
+  - [What are the most common reasons reproducibility checks fail?](#what-are-the-most-common-reasons-reproducibility-checks-fail)
+  - [What if my package uses Excel?](#what-if-my-package-uses-excel)
+
+- [Data-Related Questions](#data-related-questions)
+  - [What is original data?](#what-is-original-data)
```

Comment:
@ankritisingh this is perhaps a broader conversation but i've been using 'source data' recently as that seems to be better understood. and it's clearer to then explain that all data sources used for the analysis need to be fully documented. perhaps the FAQ here is something more like: what is the starting point for the reproducibility package? rep packages should start from documented data sources...